apps/agreement/contracts/test/mocks/disputable/AragonAppFeesCashierMock.sol

```diff
@@ -61,7 +62,7 @@ contract AragonAppFeesCashierMock is IAragonAppFeesCashier, IsContract {
     * @notice Unset fees for multiple apps
     * @param _appId App id paying for
     */
-    function payAppFees(bytes32 _appId, bytes) external {
+    function payAppFees(bytes32 _appId, bytes) external payable {
         AppFee storage appFee = appFees[_appId];
```

Comment:
This is just a mock, but I wonder if we shouldn't also handle (or revert) on the case for ETH being sent, now that it's payable?

Not a big deal, anyhow.